src/app/toolbar/toolbar.component.html

```diff
@@ -43,6 +43,11 @@
         <span class="text">Find Support</span>
         <img src="/red_ellipse.svg" />
       </a>
+      <a routerLinkActive="active-router-link" routerLink="/request-an-investigation" #findSupportLink
+        [attr.aria-current]="findSupportLink.routerLinkActive ? 'page' : false">
```

Comment:
Why are we reusing/repeating `findSupportLink`? Shouldn't this be `requestInvestigationLink` or something?